mne/io/eeglab/eeglab.py

```diff
@@ -665,7 +670,7 @@ def read_events_eeglab(eeg, event_id=None, event_id_func='strip_to_integer',
 
     annotations = _read_annotations_eeglab(eeg)
     types = annotations.description
-    latencies = annotations.onset
+    latencies = annotations.onset * eeg.srate
```

Comment:
So we always had some nasty bugs in `master` and this fixes them? An update to `whats_new.rst` documenting the changes would help guide the review

Reply:
That's not the case, here. Before, annotation onsets from `_read_annotations_eeglab` were returning latencies rather than time onsets. Now we have changed that and here we are only compensating for it. At first I added a parameter in `_read_annotations_eeglab`, add whats new and everything. But we reverted those changes in 47adee37d9 and make it silent by @agramfort suggestion. 'cos eitherway this `read_events_eeglab` must go.